### PR TITLE
donator item switch

### PR DESCRIPTION
**Pull request by CandleJaxx**
sheyehs

### CODE REVIEW

**Comment by CandleJaxx** [comment]
github is fucking drunk lmao

**Comment by CandleJaxx** [comment]
I HATE GITHUB I HATE GITHUB I HATE GITHUB

File: modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm [hunk 1]
```diff
@@ -101,11 +101,10 @@
 	item_path = /obj/item/clothing/glasses/welding/goldengoggles
 	ckeywhitelist = list("goldenalpharex")
 
-/datum/loadout_item/head/hephelmet
-	name = "HepUnit Standard Helmet"
-	item_path = /obj/item/clothing/head/helmet/sec/peacekeeper/jax
+/datum/loadout_item/shoes/funksheels
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm]:
```suggestion
/datum/loadout_item/shoes/funks_heels
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

**Comment by YakumoChen** [comment]
if this isn't an april fools joke I'm not sure if it would be mechanically fair to include a donator item that gives you a free shoes-skateboard on spawn.

**Comment by CandleJaxx** [comment]
> if this isn't an april fools joke I'm not sure if it would be mechanically fair to include a donator item that gives you a free shoes-skateboard on spawn.

you can buy them from arrivals and also: you can get a skateboard in loadout

**Comment by YakumoChen** [comment]
> > if this isn't an april fools joke I'm not sure if it would be mechanically fair to include a donator item that gives you a free shoes-skateboard on spawn.
> 
> you can buy them from arrivals and also: you can get a skateboard in loadout

wheely-heels are an uncommon maint loot exclusive item, they essentially work as no-slips and let you jump over tables, but can't be dropped and are shoe-slot, with no downsides.

**Comment by CandleJaxx** [comment]
> wheely-heels are an uncommon maint loot exclusive item, they essentially work as no-slips and let you jump over tables, but can't be dropped and are shoe-slot, with no downsides.

'exclusive' and 'essentially' are doing a lot of carry work there, as people can, again, spawn in with them roundstart with skateboards. the 'can't be dropped' is irrelevant, because if you get stunned and knocked down, you're out for a solid ten seconds without any response available. also they're heelies and it's cute as fuck

**Comment by CandleJaxx** [comment]
> 'exclusive' and 'essentially' are doing a lot of carry work there, as people can, again, spawn in with them roundstart with skateboards. the 'can't be dropped' is irrelevant, because if you get stunned and knocked down, you're out for a solid ten seconds without any response available. also they're heelies and it's cute as fuck

adding onto this, the code defines the wheelies as common, with the same weight as candle boxes. you can also win them from arcade machines

**Comment by YakumoChen** [comment]
>adding onto this, the code defines the wheelies as common, with the same weight as candle boxes. you can also win them from arcade machines

Neither are a timely guarantee to find in a shift, and yes, they are a lot more convienent than skateboards. The heels can be toggled at will from a button instead of resisting off and picking up the skateboard, and you don't lose it if you fall down, and it doesn't take up a spot in your backpack.

It's a clear mechanical advantage granted by loadout, especially as an item exclusive to one player.

**Comment by Gandalf2k15** [comment]
This is being closed for a few reasons, if you wish to discuss them further, please contact the host.

**Pull request closed without merging**